secretstores/hashicorp/vault/vault.go

```diff
@@ -209,7 +209,7 @@ func (v *vaultSecretStore) BulkGetSecret(req secretstores.BulkGetSecretRequest)
 	}
 
 	// Create list secrets url
-	vaultSecretsPathAddr := fmt.Sprintf("%s/v1/secret/data/%s", v.vaultAddress, v.vaultKVPrefix)
+	vaultSecretsPathAddr := fmt.Sprintf("%s/v1/secret/metadata/%s", v.vaultAddress, v.vaultKVPrefix)
```

Comment:
why did this change? there's another occurance for this in `getSecret`

Reply:
Mukundan made the same question. This URL here is used for the LIST API, which (as per hashicorp documentation) is a metadata operation.

https://www.vaultproject.io/api/secret/kv/kv-v2#list-secrets

In the end, I actually ran this as part of my ITs in Java SDK the old way does not work at all.